### ISSUE TITLE
[P2][Sourcing][Accessibility][p13n dialog] Initial focusing lands incorrectly while opening the p13n dialog inside <fdp-table>

### ISSUE THREAD

**Issue by yixuan-liu** [SAP/fundamental-ngx#10871]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
The initial keyboard focus either should be provided for ‘Search’ input or on the first check box has they are the important UI which needs to be selected or changed before closing this dialog.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
- Angular 15
- F-NGX Platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using the `Personalization Dialog` example under https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table,

Notice that after opening the p13n dialog by clicking on the `Settings` icon on table toolbar, the initial focus lands on the `Cancel`  button.

![Screenshot 2023-11-01 at 5 05 43 PM](https://github.com/SAP/fundamental-ngx/assets/22454655/0f9ee35d-45c9-4ad0-b292-790a5e2efcd6)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Label, information, properties of all UI elements should be conveyed correctly once to the screen reader users when accessed and additional reading should be avoided.

**Comment by droshev** [comment]
Let's move the focus to the search field instead
